SolutionAssemblyInfo.cs

```diff
@@ -16,6 +16,6 @@
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyCulture("")]
 [assembly: ComVisible(false)]
-[assembly: AssemblyVersion("1.8.0")]
-[assembly: AssemblyFileVersion("1.8.0")]
-[assembly: AssemblyInformationalVersion("1.8.0")]
\ No newline at end of file
+[assembly: AssemblyVersion("1.0.0")]
+[assembly: AssemblyFileVersion("1.0.0")]
+[assembly: AssemblyInformationalVersion("1.0.0")]
```

Comment:
Why these has been changed to 1.0.0?

Reply:
AppVeyor patches these properties at build time. I changed them so they don't get confusing and because we don't need to keep updating them, thought I set them as a base version

Reply:
once you approve this you will need to also reapprove release pr, and then roll out 1.8.0

Reply:
@jjw24 This changes break the Debug config build on my machine, revert back to 1.8.0 seems to fix, do you have the same problem?

```
System.IO.FileNotFoundException: Could not load file or assembly 'Flow.Launcher.Infrastructure, Version=1.1.2.0, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.
File name: 'Flow.Launcher.Infrastructure, Version=1.1.2.0, Culture=neutral, PublicKeyToken=null'
   at System.Reflection.RuntimeAssembly.GetExportedTypes(QCallAssembly assembly, ObjectHandleOnStack retTypes)
   at System.Reflection.RuntimeAssembly.GetExportedTypes()
   at System.Reflection.Assembly.get_ExportedTypes()
   at Flow.Launcher.Core.Plugin.PluginAssemblyLoader.FromAssemblyGetTypeOfInterface(Assembly assembly, Type type) in C:\data\common\nextcloud\projects\python\Flow.Launcher\Flow.Launcher.Core\Plugin\PluginAssemblyLoader.cs:line 58
   at Flow.Launcher.Core.Plugin.PluginsLoader.<>c__DisplayClass2_1.<DotNetPlugins>b__1() in C:\data\common\nextcloud\projects\python\Flow.Launcher\Flow.Launcher.Core\Plugin\PluginsLoader.cs:line 47
   at Flow.Launcher.Infrastructure.Stopwatch.Debug(String message, Action action) in C:\data\common\nextcloud\projects\python\Flow.Launcher\Flow.Launcher.Infrastructure\Stopwatch.cs:line 19
   at Flow.Launcher.Core.Plugin.PluginsLoader.DotNetPlugins(List`1 source) in C:\data\common\nextcloud\projects\python\Flow.Launcher\Flow.Launcher.Core\Plugin\PluginsLoader.cs:line 41
   at Flow.Launcher.Core.Plugin.PluginsLoader.Plugins(List`1 metadatas, PluginsSettings settings) in C:\data\common\nextcloud\projects\python\Flow.Launcher\Flow.Launcher.Core\Plugin\PluginsLoader.cs:line 25
   at Flow.Launcher.Core.Plugin.PluginManager.LoadPlugins(PluginsSettings settings) in C:\data\common\nextcloud\projects\python\Flow.Launcher\Flow.Launcher.Core\Plugin\PluginManager.cs:line 105
   at Flow.Launcher.App.<OnStartupAsync>b__14_0() in C:\data\common\nextcloud\projects\python\Flow.Launcher\Flow.Launcher\App.xaml.cs:line 69
   at Flow.Launcher.Infrastructure.Stopwatch.NormalAsync(String message, Func`1 action) in C:\data\common\nextcloud\projects\python\Flow.Launcher\Flow.Launcher.Infrastructure\Stopwatch.cs:line 58
   at Flow.Launcher.App.OnStartupAsync(Object sender, StartupEventArgs e) in C:\data\common\nextcloud\projects\python\Flow.Launcher\Flow.Launcher\App.xaml.cs:line 50
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__140_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.WrappedInvoke(Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)
   at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.WrappedInvoke(Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at System.Windows.Threading.Dispatcher.Invoke(DispatcherPriority priority, Delegate method, Object arg)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.TranslateAndDispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.Run()
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at System.Windows.Application.Run(Window window)
   at System.Windows.Application.Run()
   at Flow.Launcher.App.Main() in C:\data\common\nextcloud\projects\python\Flow.Launcher\Flow.Launcher\App.xaml.cs:line 43
```

Reply:
Strange. I have not built locally after this pr, but have done the same change numerous times previously to test prs. Im afk atm will build and let you know later.

Have you tried cleaning and regenerate the nuget packages and remove external plugins to test?

Reply:
I did clean solution but not touch nuget and external plugins stuff yet, I can try those later 😄

Reply:
That's from Everything plugin.

Reply:
@pc223 just done a rebuild locally and runs fine. are you still having this issue? can you check if it's everything plugin

Reply:
> @pc223 just done a rebuild locally and runs fine. are you still having this issue? can you check if it's everything plugin

Yeah, seems it breaks when I install the Everything plugin, not sure why

@taooceros how do you know? I see nothing in the logs 🧐

Reply:
> > @pc223 just done a rebuild locally and runs fine. are you still having this issue? can you check if it's everything plugin
> 
> Yeah, seems it breaks when I install the Everything plugin, not sure why
> 
> @taooceros how do you know? I see nothing in the logs 🧐 

The exception is unable to load infrastructure assembly, but it doesn't crash flow when I don't attach debugger. Only everything plugin include a legacy infrastructure dll, which may cause the crash. So I try deleting it, and it is the reason.

Probably this is because the version of infrastructure dll in everything plugin is higher than 1.0.0.0 in Flow, which cause the assembly not found issue.